yarn.lock

```diff
@@ -1372,9 +1372,9 @@
     loader-utils "^2.0.0"
 
 "@tryghost/content-api@^1.5.2":
-  version "1.9.8"
-  resolved "https://registry.yarnpkg.com/@tryghost/content-api/-/content-api-1.9.8.tgz#91a83e8baf52705a01d2fe99b07f90d64a066ff2"
-  integrity sha512-aBhEjd5bAA4iNQqX5azhYJ+Ln9w3SnduxfVi1QyS6qDT9WqvTSxBzkKmCARFmfxmlW6o7l+90qmpOtysEqlsOw==
+  version "1.11.0"
```

Comment:
Was there a package.json change that prompted this?

Reply:
Yes, I (think, it's been a while) ran into an error about user-agents and bumping ghost's content-api turned up as a suggestion (probably on Stack Overflow) to resolve it.

the relevant code is https://github.com/DemocracyLab/CivicTechExchange/pull/903/commits/a3abf6bfd21262bbbfef326c684bf36cba54cf66 -- On review of that commit, it seems like passing an explicit limit to the number of returned posts was the change that prompted this. But like I said -- it's been a while. If you want specifics let me know, I can roll the version back and re-test.

Reply:
If we need the change, that's fine, but we should add it to package.json or else your changes will just get overwritten the next time package.json changes.

Reply:
this is what I get for looking at PRs in the evening. Didn't even catch that it wasn't in package.json yet. Fixed that.